src/Resources/app/administration/src/module/findologic-module/components/findologic-config/findologic-config.html.twig

```diff
@@ -1,217 +1,106 @@
 {% block findologic_credentials %}
-<sw-container slot="grid" rows="auto auto">
-    <sw-card-section>
-        <sw-card class="sw-card--grid"
-                 :title="$tc('findologic.settingForm.config.title')">
-            <sw-container>
-                {% block findologic_credentials_card_container %}
-
-                    {% block findologic_credentials_settings %}
-                        <div class="findologic-settings-credentials-fields" v-if="actualConfigData">
-                            {% block findologic_credentials_settings_shopkey %}
-                                <sw-inherit-wrapper
+    <sw-container slot="grid" rows="auto auto">
+        <sw-card-section>
+            <sw-card class="sw-card--grid"
+                     :title="$tc('findologic.settingForm.config.title')">
+                <sw-container>
+                    {% block findologic_credentials_card_container %}
+                        {% block findologic_credentials_settings %}
+                            <div class="findologic-settings-credentials-fields">
+                                <sw-text-field
```

Comment:
Seems like all previous issues regarding the configuration page have been solved, just one minor thing is, that when removing the shopkey, I can save the configuration properly, but it still tells me that it is required. This should not be the case.

![image](https://user-images.githubusercontent.com/20555112/100594596-2b9f3900-32fa-11eb-8def-8b00d514f02e.png)